### PR TITLE
Add Comparison and use cases

### PR DESCRIPTION
**Pull request by auxten**

### CODE REVIEW

**Review by laodouya — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -67,8 +110,45 @@ sql.Open("CovenantSQL", dbURI)
...
+2. CQL can record SQL query records on the blockchain, which is very convenient for customers to check their bills for long-tail customers and billing, like [this](https://explorer.dbhub.org/dbs/7a51191ae06afa22595b3904dc558d41057a279393b22650a95a3fc610e1e2df/requests/f466f7bf89d4dd1ece7849ef3cbe5c619c2e6e793c65b31966dbe4c7db0bb072)
+3. For customers with high performance requirements, Slave nodes can be deployed at the customer to meet the needs of customers with low latency queries while enabling almost real-time data updates.
+
+#### Secure storage
+
+Thanks to the CQL data history is immutable, CQL can be used as a storage for sensitive operational logs to prevent hacking and erasure access logs.
+
+### ĐApp
+
+Storing data on Bitcoin or Ethereum is quite expensive ($4305 / MB on Ethereum 2018-05-15). Programming is very complicated due to the lack of support for structured data storage. CQL gives you a low-cost structured SQL database also provides more room for ĐApp to exchange data with real-world.
```
Comment (leventeliu) [inline README.md]: CQL gives you a low-cost structured SQL database **and** also...

Comment (leventeliu) [reply README.md]: And there's trailing space here.

File: README.md [hunk 2]
```diff
@@ -57,8 +70,38 @@ sql.Open("CovenantSQL", dbURI)
...
+
+#### FootNotes
+
+- <a name="bft-raft">BFT-Raft</a>: A CQL leader offline needs CQL Block Producer to decide whether to wait for leader online for data integrity or promote a follower node for availability. This part is still under construction and any advice is welcome.  
+- <a name="transaction">Transaction</a>: Talking about `ACID`, CQL has full "Consistency, Isolation, Durability" and a limited `Atomicity` support. That is even under strong consistency mode, CQL transaction is only supported on the leader node. If you want to do "read `v`, `v++`, write `v` back" parallelly and atomically, then the only way is "read `v` from the leader, `v++`, write `v` back to leader"
+- <a name="fuse">FUSE</a>: CQL has a [simple FUSE](https://github.com/CovenantSQL/CovenantSQL/tree/develop/cmd/cql-fuse) support adopted from CockroachDB. The performance is not very ideal and still has some issues. But it can pass fio test like `fio --debug=io --loops=1 --size=8m --filename=../mnt/fiotest.tmp --stonewall --direct=1 --name=Seqread --bs=128k --rw=read --name=Seqwrite --bs=128k --rw=write --name=4krandread --bs=4k --rw=randread --name=4krandwrite --bs=4k --rw=randwrite`
```
Comment (leventeliu) [inline README.md]: Wrap long command in a code block.

File: README.md [hunk 3]
```diff
@@ -57,8 +70,38 @@ sql.Open("CovenantSQL", dbURI)
...
+2. BFT-Raft (Byzantine Fault-Toleranted Raft)<sup>[bft-raft](#bft-raft)</sup> is applied in `Strong consistency mode` database. We named our implementation [`Kayak`](https://github.com/CovenantSQL/CovenantSQL/tree/develop/kayak).  The CQL miner leader does a `Two-Phase Commit` with `Kayak` to support `Transaction`.<sup>[transaction](#transaction)</sup>
+
+CQL database consistency mode and node count can be selected in datebase creation with command  `cql create '{"UseEventualConsistency": true, "Node": 3}'`
+
+## Comparison
+
+|                              | Ethereum          | Hyperledger Fabric       | Amazon QLDB   | CovenantSQL                                                  |
+| ---------------------------- | ----------------- | ------------------------ | ------------- | ------------------------------------------------------------ |
+| **Dev language**             | Solidity          | Chaincode   (Go, NodeJS) | ?             | Python, Golang, Java, PHP, NodeJS, MatLab                    |
```
Comment (foreseaz) [inline README.md]: [Ethereum flavored WebAssembly]( https://github.com/ewasm/design) could add to Ethereum smart contract dev language.

**Review by leventeliu — APPROVED** [review]
LGTM.

**Review by foreseaz — APPROVED** [review]
LGTM

**Pull request merged**